docs/G-Codes.md

```diff
@@ -1313,6 +1313,20 @@ print.
 #### SDCARD_RESET_FILE
 `SDCARD_RESET_FILE`: Unload file and clear SD state.
 
+### [axis_twist_compensation]
+
+The following commands are available when the
+[axis_twist_compensation config
+section](Config_Reference.md#axis_twist_compensation) is enabled.
+
+#### AXIS_TWIST_COMPENSATION_CALIBRATE
+`AXIS_TWIST_COMPENSATION_CALIBRATE [N_POINTS=<value>]`: Initiates the X twist calibration
```

Comment:
I'd prefer to avoid abbreviations in the parameter names, so instead of `N_POINTS` maybe something like `POINTS` or `SAMPLE_COUNT` or `SAMPLES` or `PROBE_COUNT` or similar.

Reply:
I would prefer `SAMPLE_COUNT`, but the others are fine too.